lib/types/multipart.js

```diff
@@ -199,6 +199,8 @@ function Multipart (boy, cfg) {
             file.truncated = true
             part.removeAllListeners('data')
           } else if (!file.push(data)) { self._pause = true }
+
+          file.bytesRead = nsize > fileSizeLimit ? fileSizeLimit : nsize
```

Comment:
does this cause any noticeable perf impact?

Reply:
with the current busboy benchmarks, no.

But we can wait till you have created more reliable benchmarks.  :)

Reply:
@kibertoad  Can you please check if this has some relevant performance impact?

Reply:
will do!